content/sensu-go/6.3/observability-pipeline/observe-schedule/service-components.md

```diff
@@ -0,0 +1,547 @@
+{{< code json >}}
+{
+  "rules": [
+    {
+      "arguments": {
+        "status": "non-zero",
+        "threshold": 25
+      },
+      "name": "nonzero-25",
+      "template": "status-threshold"
+    }
+  ]
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+services     | 
+-------------|------ 
+description  | List of business service entities that include the service component as a dependency.
+required     | true
+type         | Array
+example      | {{< language-toggle >}}
+{{< code yml >}}
+services:
+  - account-manager
+  - tessen
+{{< /code >}}
+{{< code json >}}
+{
+  "spec": {
+    "services": [
+      "account-manager",
+      "tessen"
+    ]
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### Query attributes
+
+type         | 
+-------------|------ 
+description  | Type of selector to use to identify the events that the service component's monitoring rule should process: `fieldSelector` or `labelSelector`.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+type: labelSelector
+{{< /code >}}
+{{< code json >}}
+{
+  "type": "labelSelector"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+value        | 
+-------------|------ 
+description  | Selector expression the query will use to identify the events that the service component's monitoring rule should process.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+value: region == us-west-2 && cmpt == psql
+{{< /code >}}
+{{< code json >}}
+{
+  "value": "region == us-west-2 && cmpt == psql"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### Rules attributes
+
+arguments    | 
+-------------|------ 
+description  | The [arguments][5] to pass to the [rule template][8] for the service component.
+required     | false
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+arguments:
+  status: non-zero
+  threshold: 25
+{{< /code >}}
+{{< code json >}}
+{
+  "arguments": {
+    "status": "non-zero",
+    "threshold": 25
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+name         | 
+-------------|------ 
+description  | Explicit name to use for the rule-specific events generated for the service component. These names help keep events distinct when a service component includes different rules for the same [rule template][8].
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+name: nonzero-25
+{{< /code >}}
+{{< code json >}}
+{
+  "name": "nonzero-25"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+template     | 
+-------------|------ 
+description  | Name of the [rule template][8] the service component should use.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+template: status-threshold
+{{< /code >}}
+{{< code json >}}
+{
+  "template": "status-threshold"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+##### Arguments attributes
+
```

Comment:
Do the arguments attributes change depending on the rule template? In other words, can rule templates use different attributes than status and threshold?